### DISCUSSION TITLE
Permit Users to Choose Their Install Directory

### DISCUSSION THREAD

**Discussion by iamhumanipromise** [Squirrel/Squirrel.Windows#1882] [Ideas]
Squirrel is beautiful in that the default behavior is to install without interaction. However; what about the edge cases where users have a prod raid and a test raid (database developers. medical industry employees) and local, manual intervention is desired for choosing the directory.

**Comment by anaisbetts** [comment]
I recommend changing the NuGet package ID for your test builds which will change the installation directory
